packages/actions/src/Concerns/CanImportRecords.php

```diff
@@ -385,6 +413,11 @@ public function getMaxRows(): ?int
         return $this->evaluate($this->maxRows);
     }
 
+    public function getDelimiter(): ?string
+    {
+        return $this->evaluate($this->csvDelimiter);
```

Comment:
I had considered including some sort of validation here (ie: if something other than the allowed 1-char string gets set), but decided that letting LeagueCSV throw an exception makes best sense: the dev should discover it in testing since the delimiter is set in code.

Reply:
Agree